modules/402-ingress-nginx/templates/controller/secret.yaml

```diff
@@ -9,11 +9,6 @@ metadata:
   namespace: d8-ingress-nginx
   {{- include "helm_lib_module_labels" (list $context ) | nindent 2 }}
 data:
-  client.crt: {{ $cert.data.certificate | b64enc }}
+  client.crt: {{ $cert.data.cert | b64enc }}
   client.key: {{ $cert.data.key | b64enc }}
-
-{{- /* By deploying and deleting this ingress we trigger "nginx reload". It is required to update client certififcates. */ -}}
-  {{ if $cert.data.certificate_updated }}
```

Comment:
Are you sure we don't need it?

Reply:
There is no `certificate updated` field. I think we will need it in the future. As for now, certificates are issued for ten years.